packages/packages/core/editor-components/src/prop-types/component-overridable-prop-type.ts

```diff
@@ -5,6 +5,11 @@ export const componentOverridablePropTypeUtil = createPropUtils(
 	'component-overridable',
```

Comment:
(Have been changed in PHP 🥲 )
```suggestion
	'overridable',
```